jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -95,6 +96,8 @@ when "forward_only"
   xfcc_forward_only = true
 when "sanitize_set"
   xfcc_sanitize_set = true
+when "forward_only_if_route_service"
+  xfcc_forward_only_if_route_service = true
 else
   abort("Unkown 'forwarded_client_cert' option: #{forwarded_client_cert_option}. Known options: 'always_forward_only', 'forward_only', 'sanitize_set'")
```

Comment:
we should add `xfcc_forward_only_if_route_service ` to the list of possible options here